templates/cisco_ios_show_adjacency_detail.textfsm

```diff
@@ -0,0 +1,36 @@
+Value Required INTERFACE (\S+)
+Value Required ENDPOINT (\S+)
+Value List REWRITE_HEADERS ([A-F0-9]+)
+Value RECURSIVE_INTERFACE (\S+)
+Value RECURSIVE_NEXTHOP (\S+)
+
+Start
+  ^Protocol\s+Interface\s+Address\s*$$ -> Entries
```

Comment:
Do you mind adding the following to each state? You have the error for two, but not the blank line.

```
  ^\s*$$
  ^. -> Error
```

Everything looks good besides those minor things.

Reply:
Thanks for your feedback!
Does it include 'Start' state as well?
How do I push new changes (sorry I am reasonably new to GitHub)? Do I just commit in my branch, then push to my forked copy and then do something within GitHub?
Regards